bin/react-scripts.js

```diff
@@ -5,14 +5,17 @@ var args = process.argv.slice(3);
 
 switch (script) {
 case 'build':
-case 'start':
 case 'eject':
+case 'start':
   spawn(
     'node',
     [require.resolve('../scripts/' + script)].concat(args),
     {stdio: 'inherit'}
   );
   break;
+case 'test':
+  require('jest/bin/jest');
```

Comment:
>Inspect your code changes or re-run with `-u` to update them.

This will be confusing because people will try `npm test -u`.
I don’t have a better solution though.